homeassistant/helpers/script.py

```diff
@@ -1212,13 +1212,8 @@ async def async_run(
             raise
 
     async def _async_stop(
-        self, update_state: bool, spare: _ScriptRun | None = None
+        self, aws: list[asyncio.Task], update_state: bool, spare: _ScriptRun | None
```

Comment:
We removed the default `None` value for the `spare` parameter. Is that intended?

Reply:
Yes, because `_async_stop` is only called from `async_stop` which just passes on the `spare` parameter.